src/data/local/github/tracks-node/track-4/steps/step-1.md

```diff
@@ -1,33 +1,38 @@
-In this track, you will implement a login mutation fully integrated with the database. There are many ways of implementing an [authentication](https://en.wikipedia.org/wiki/Authentication_server). The way we're doing it is: the user sends a password to prove that he is who he claims, and then the server provides a token that allows him to access some data on future requests. This token generally have an expiration, after which the user has to login again to get a new one. 
-
 # Step 1/4 - The mutation setup
 ### Estimated time: 1 hour
 
-Let's start with baby steps, by creating the mutation prototype, with no integration with the dabatase (yet).
+On this track, you're going to develop a mutation to create users on database. Let's start with baby steps, by creating the mutation prototype, with no integration to the dabatase (yet). The main purpose of this step is to exercise only how to create a mutation, a type and an input type on GraphQL Schema.
 
-Let's call the mutation `login`. It should receive an e-mail and a password as parameters and, in case of success, return the following structure on body:
+Let's call the mutation `createUser`. It will receive an object of the type `UserInput`, which you can call `data`, with the following fields:
 
 ```json
 {
   "data": {
-    "login": {
-      "user": {
-        "id": "12",
-        "name": "User Name",
-        "email": "User e-mail",
-        "birthDate": "04-25-1990",
-        "cpf": "XXXXXXXXXXX",
-      },
-      "token": "the_token"
-    }
+    "name": "User Name",
+    "email": "User e-mail",
+    "password": "user password",
+    "birthDate": "01-01-1990"
   } 
 }
 ```
 
-Your `User` can have other information if you want, but these fields above should be enough.
+The response will be almost the same type as the input, with the adittion of an `id` field. This will be the primary key of the User table on database. Also, we souldn't return `password` field 🔑, but the GraphQL Schema should get it covered for us. Thanks, GraphQL!
```

Comment:
```suggestion
The response will be almost the same type as the input, with the adittion of an `id` field (for now, this can be a random number). This will be the primary key of the User table on database. Also, we souldn't return `password` field 🔑, but the GraphQL Schema should get it covered for us. Thanks, GraphQL!
```
Ou algo do gênero, só pra garantir (já que ainda não tem integração com o banco)